### PR TITLE
IDIR Dashboard

### PR DESCRIPTION
**Pull request by amichard**
#1475 #1476 #1477 #1478 #1479 #1480 #1481

Changelog:
- IDIR users should now be redirected to the dashboard by default
- bceid users will still be redirected to credit transactions
- Nav Bar has been updated to match dashboard
- Numbers in dashboard are dynamic (not hard-coded)

### CODE REVIEW

**Comment by amichard** [comment]
[Dashboard | BCEID/IDIR Dashboard panel User Settings |1|](https://trello.com/c/H4dbnZwP/1603-dashboard-bceid-idir-dashboard-panel-user-settings-1)

**Comment by amichard** [comment]
[Dashboard | IDIR Dashboard |1|](https://trello.com/c/W9LIwB9X/1594-dashboard-idir-dashboard-1)

**Comment by amichard** [comment]
[Dashboard | IDIR Dashboard metric all org credit balance |1|](https://trello.com/c/xbt7BrN2/1604-dashboard-idir-dashboard-metric-all-org-credit-balance-1)

**Comment by amichard** [comment]
[Dashboard | IDIR Dashboard panel Credit Transactions |2|](https://trello.com/c/D9zDGD0j/1607-dashboard-idir-dashboard-panel-credit-transactions-2)

**Comment by amichard** [comment]
[IDIR Dashboard panel Compliance & Exclusion Reports​ |2|](https://trello.com/c/Zv9N0ygA/1608-idir-dashboard-panel-compliance-exclusion-reports-2)

**Comment by amichard** [comment]
[Dashboard | IDIR Dashboard panel Fuel Codes​ |2|](https://trello.com/c/HIDd78Gf/1609-dashboard-idir-dashboard-panel-fuel-codes-2)

**Comment by amichard** [comment]
[IDIR Dashboard panel Administration |1|](https://trello.com/c/d7coYauG/1611-idir-dashboard-panel-administration-1)

File: frontend/src/compliance_reporting/components/ComplianceReportingTable.js [hunk 1]
```diff
@@ -33,7 +32,45 @@ const ComplianceReportingTable = (props) => {
     id: 'type',
     minWidth: 75
   }, {
-    accessor: item => <ComplianceReportStatus status={item.status} />,
+    accessor: (item) => {
```
Comment (plasticviking) [inline frontend/src/compliance_reporting/components/ComplianceReportingTable.js]: Should this be factored out into a helper?

Comment (amichard) [reply frontend/src/compliance_reporting/components/ComplianceReportingTable.js]: Sure

**Pull request merged**